### ISSUE TITLE
Impossible to run decoding without reference

### ISSUE THREAD

**Issue by neubig** [neulab/xnmt#390]
Currently evaluation allows you to decode and evaluate against a reference, but it seems that there's no way to perform decoding without a reference. We should probably add this?

**Comment by msperber** [comment]
Yeah, I would need that feature also. Should this simply be a new commandline tool ```xnmt_decode```, or would it be useful if it were also specifiable in the YAML config files?

**Comment by neubig** [comment]
We could add a new task like PredictionTask or DecodingTask in eval_task.py
maybe.

**Comment by neubig** [comment]
Oh, and independently of this, I think it would be nice to have it from the command line: https://github.com/neulab/xnmt/issues/290
